docs/source/conf.py

```diff
@@ -55,7 +55,7 @@
 
 # General information about the project.
 project = u'conda'
-copyright = u'2017, Anaconda, Inc.'
+copyright = u'2023, Anaconda, Inc.'
```

Comment:
The copyright doesn't need to be updated here. That's when the docs were created and it's not that useful (and probably not even correct, legally speaking) to update that every year.

Reply:
Changed it back. Thanks!